anchore-policy-validator/README-dev.md

```diff
@@ -0,0 +1,19 @@
+# Dev helper
+
+Due to some resource create with specific deployment method (eg.: create via k8s job), resources has to be deleted by manually.
```

Comment:
`some resource` -> `some resources`
`to be deleted by manually` -> `to be deleted manually`
Although, these can be solved using helm hooks as well (run delete jobs as part of helm delete).

Reply:
jobs are deleted with helm hooks, but resources, which are created with these jobs, are cannot deleted via helm hooks